other/postgres-entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [[ "${DB_BACKUP}" == "true" ]]; then
+    cat > /etc/periodic/daily/backup <<EOF
+#!/bin/sh
+pg_dump -d $POSTGRES_DB -U $POSTGRES_USER -F t -f /tmp/db.tar > /tmp/db.log 2>&1
+EOF
+fi
+
+re='^[0-9]+$'
+if [[ $DB_AUTOCLEAN =~ $re ]]; then
```

Comment:
If someone does not want autoclean he simply puts a None value there?

Reply:
please also add a default value (e.g., None) in the current .env file

Reply:
default will be none? now its 24